Gemfile

```diff
@@ -36,12 +36,19 @@ gem 'jbuilder', '~> 2.5'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+# Gems que tuve que agregar para la app que estoy haciendo
 # Rubocop -> Me lo pidieron en una card
 gem 'rubocop', '~> 0.59.1', require: false
 
 # Devise para el login
 gem 'devise'
 
+# Autenticacion de logins
+gem 'devise_token_auth'
+
+# Esta gem me la pidio devise_token_auth
+gem 'omniauth'
```

Comment:
Normaly, you do not need this gem. This issue from devise_auth_token is resolved seen 2 years.